compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -966,9 +966,11 @@ TR::Register *OMR::Power::TreeEvaluator::istoreEvaluator(TR::Node *node, TR::Cod
       }
 
    bool reverseStore = false;
-   if (valueChild->getOpCodeValue() == TR::ibyteswap)
+   if (valueChild->getOpCodeValue() == TR::ibyteswap && !valueChild->getRegister() && valueChild->getReferenceCount() == 1)
```

Comment:
We actually have a node API `isSingleRefUnevaluated` for this:
https://github.com/eclipse/omr/blob/d4d9bd5c8c04d8f9d136aa94716d953bac424217/compiler/il/OMRNode_inlines.hpp#L143-L148

Could we use it instead?

Reply:
Huh, I didn't know we had an API like this but that seems to make checking this much easier. AFAICT that API is only currently only used on Z. I've updated this PR to use it, but perhaps we should open a issue to go through and update the other codegens to use it where appropriate?

Reply:
I agree, we should start using this API more. Do you want to open up such an issue? We also should write some Doxygen documentation for it.

Reply:
Opened #5648 for this.